app/assets/images/payment_icons/latitude_go_au.svg

```diff
@@ -0,0 +1 @@
+<svg height="24" viewBox="0 0 38 24" width="38" role="img" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-labelledby="pi-latitude_go_au"><title id="pi-latitude_go_au">Latitude GO (AU)</title>><clipPath><path clip-rule="evenodd" d="M35 0H3a2.946 2.946 0 00-3 3v18a3.009 3.009 0 003 3h32a2.946 2.946 0 003-3V3a3.009 3.009 0 00-3-3z"/></clipPath><g clip-path="url(#a)"><image height="138" transform="matrix(.24 0 0 .24 -.16 -8.12)" width="159" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJ8AAACKCAMAAACD3mj6AAAAolBMVEXnfSfyyQrWJyrbR0/pqL/faHXUJyqzJmioJXyzIn3TGH+qLIGAP4I8NWc8MGaALX1pLnUlMV47RnN8hLNha5p7grCRkLKYlLIAiLRGibGLiq4Li7depsKwwMwkfatbn76RwNB2r8cAjbwbe7U1aa5jh42x00oAp6kAoo8oq4GfxFhHrVcqpVeCuUGfwzH04QLx4Avn3iSuxzbZ2CnGzEPX1TTe/uAOAAAAoklEQVR42u3Ouw7CMBAEwF3F///HoBQHxBISBVWKqby+l6ZJ2mSednJ7ldP2+mzpPfxpTKkTt+k+Vr7u7Yt99bodbtJnjvLx8fHx8fHx8fHx8fHx8fHx8fH961s394WPj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+P77fvBNjRL2W3Jy/AAAAAAElFTkSuQmCC"/><path d="M-.693 4.777h15.657a1.657 1.657 0 011.658 1.657v11.05a1.657 1.657 0 01-1.658 1.657H-.693z" fill="#0046aa"/></g><path d="M0 3v18a3.009 3.009 0 003 3h32a2.946 2.946 0 003-3V3a3.009 3.009 0 00-3-3H3a2.946 2.946 0 00-3 3zm35-2a2.006 2.006 0 012 2v18a2.006 2.006 0 01-2 2H3a2.006 2.006 0 01-2-2V3a2.006 2.006 0 012-2z" fill="#002a66" fill-rule="evenodd" opacity=".3"/><g fill="#fff"><circle cx="8.358" cy="8.895" r=".663"/><path d="M10.441 9.558a.663.663 0 10-.663-.658.663.663 0 00.663.658zM13.187 15.809v-1.326h-6.25v-6.25H5.611v7.576zM12.525 9.558a.663.663 0 10-.663-.663.662.662 0 00.663.663z"/><path d="M12.525 13.725a.663.663 0 10-.663-.663.662.662 0 00.663.663zM12.525 11.642a.663.663 0 10-.663-.663.662.662 0 00.663.663z"/><path d="M26.963 9.985a1.974 1.974 0 101.983 1.974 1.929 1.929 0 00-1.983-1.974zm-5.594-.653a2.565 2.565 0 011.968.793l-.454.455a2.026 2.026 0 00-1.485-.6 1.975 1.975 0 10-.007 3.948 2.159 2.159 0 001.2-.331v-.968h-.975v-.7h1.678v2a3.027 3.027 0 01-1.939.653 2.629 2.629 0 11.015-5.254zm5.594 0a2.628 2.628 0 11-2.729 2.627 2.6 2.6 0 012.729-2.627z" fill-rule="evenodd"/></g></svg>
```

Comment:
Same for this one

[![alt](https://screenshot.click/08-20-88ezv-qhp3r.png)](https://screenshot.click/08-20-88ezv-qhp3r.png)